tests/opwen_email_server/integration/test_celery.py

```diff
@@ -3,38 +3,80 @@
 from string import ascii_letters
 from unittest import TestCase
 from unittest import skipUnless
+import ast
+import inspect
 
 from cached_property import cached_property
 from kombu import Connection
 from kombu import Exchange
 from kombu import Queue
 
 from opwen_email_server.config import QUEUE_BROKER
+import opwen_email_server.integration.celery as celery
```

Comment:
I believe this can be simplified to `from opwen_email_server.integration import celery`.